alacritty_terminal/src/event.rs

```diff
@@ -99,6 +99,7 @@ pub trait EventListener {
     fn send_event(&self, _event: Event) {}
 }
 
-/// Placeholder implementation for tests.
-#[cfg(test)]
```

Comment:
Having it as `cfg(test)` prevents `alacritty` crate for using it for tests. So I've added a dummy type pure for tests, so we can use `mock_term`. Though, I can move it to `alacritty_terminal::term::tests` so it'll be more clear what it used for.